telemetry/specs/INT.mdk

```diff
@@ -788,9 +782,8 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   - bit3: Queue ID + Queue occupancy
   - bit4: Ingress timestamp
   - bit5: Egress timestamp
-  - bit6: Queue ID + Queue congestion status
+  - bit6: Level 2 Ingress Port ID + Egress Port ID (4 bytes each)
   - bit7: Egress port tx utilization
-  - bit8: Level 2 Ingress Port ID + Egress Port ID (4 bytes each)
   - bit15: Checksum Complement
   - The remaining bits are reserved.
   Each instruction requests 4 bytes of metadata to be inserted at each hop,
```

Comment:
"except if bit 8 is set" in next line should now change to "except if bit 6 is set"

Reply:
thanks, done.